source/graphql/migrate-hasura.txt

```diff
@@ -0,0 +1,263 @@
+.. _migrate-hasura:
+
+Migrate GraphQL to Hasura
+=========================
+
+.. meta::
+  :description: Learn how to migrate your GraphQL host from Atlas App Services to Hasura.
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. important::
+
+   Always refer to the official documentation of both MongoDB Atlas and `Hasura 
+   <https://hasura.io/docs/latest/databases/mongodb/mongodb-atlas/>`_
+   for the most up-to-date and accurate information. Specific steps may vary
+   depending on the details of your project and the technologies
+   used.
+
+Hasura empowers developers to rapidly build and deploy GraphQL and `REST APIs
+on MongoDB <https://hasura.io/graphql/database/mongodb>`_ and many other data
+sources. By radically cutting down API development times, Hasura enables rapid
+access to data, reduces friction across teams and services, and enables
+enterprises to shorten time to market on data-powered products and features.
+
+Before You Begin
+----------------
+
+If you haven't already, Create an account on Hasura's website `cloud.hasura.io 
+<https://cloud.hasura.io/signup?utm_source=mongodb&utm_medium=blog&utm_campaign=mongodb_developer_portal>`_.
```

Comment:
Just a Q - are we ok with adding a utm link for somebody else to our docs? Do we have any policies on this as an org?

As a user, I personally hate tracking links, and if I realized this was a tracking link that isn't even owned by the domain I'm visiting, I'd probably be upset by this.

Also, I would _think_ there could be legal ramifications here around privacy policy? As a MongoDB customer, I may have read/agreed to MongoDB's privacy policy, but I'm not necessarily aware of this link involving privacy data for a privacy policy I haven't even seen yet for a third-party.

Reply:
It does feel a bit icky to have links like this in docs. But, this info could help strengthen MongoDB's relationship with an important partner.

I don't think it's a privacy concern, at least, not a legal one. It's 100% anonymous and is only a URL query parameter - unless my understanding of how UTM works is way off. UTMs are generally considered GDPR compliant.